collections/river-and-lake-ice-extent-s2.yaml

```diff
@@ -0,0 +1,74 @@
+Name: Sentinel 2 River and Lake Ice Extent
+Description: |
+  The Sentinel 2 River and Lake Ice Extent (RLIE S2) product is part of the Copernicus Land Monitoring Service (CLMS), 
+  pan-European High Resolution Snow and Ice Monitoring [(HR-S&I)](https://land.copernicus.eu/pan-european/biophysical-parameters/high-resolution-snow-and-ice-monitoring) Ice product suite.The River and Lake Ice Extent (RLIE) products measure the presence of ice in rivers and lakes described by the EU-HYDRO network database, in particular the area of snow-free or snow-covered ice. 
+  The RLIE S2 product is released in near real time for the entire EEA39 zone based on the revisit time of the Sentinel 2 constellation, and it is generated at a spatial resolution of 20 m x 20 m.  Because it is time critical for several applications, every effort is made to provide near real time ice products between 6 and 12 hours after the data sensing time. 
+  More information about the RLIE S2 product methodology and workflow is outlined in the [product user manual](https://land.copernicus.eu/user-corner/technical-library/hrsi-ice-pum)
+AdditionalInfoExternal:
+    Title: Additional info 
+    Path: river-and-lake-ice-extent-s2/README.MD 
+Image: river-and-lake-ice-extent-s2/thumbnail.png       
+Resolution: 20m
+GeographicalCoverage: Europe  (Lat; 26N 72N Lon; -25W 45E)
+TemporalAvailability: September 1, 2016 - ongoing 
+UpdateFrequency: Annually
```

Comment:
Should be the same as in the description

`NRT, between 6 and 12 hours after the data sensing time.`